frontend/src/components/CensusDataChart/CensusDataChart.tsx

```diff
@@ -0,0 +1,61 @@
+import 'chart.piecelabel.js'
+import 'chartjs-plugin-stacked100'
+import * as React from 'react'
+import { HorizontalBar } from 'react-chartjs-2'
+import { ADEQUACY_COLORS } from '../../constants/colors'
+import { AdequacyMode, Method } from '../../constants/datatypes'
+import { StatisticsByGroup } from '../../utils/data'
+import { getLegend } from '../MapLegend/MapLegend'
+
+type Props = {
+    percent: boolean,
+    measurementMethod: Method,
+    censusGroups: string[],
+    populationByAdequacyByGroup: StatisticsByGroup
+}
+
+export let CensusDataChart: React.StatelessComponent<Props> = ({ percent, measurementMethod, censusGroups, populationByAdequacyByGroup }) => {
+    let adequacyModes = [AdequacyMode.ADEQUATE_15, AdequacyMode.ADEQUATE_30, AdequacyMode.ADEQUATE_60, AdequacyMode.INADEQUATE]
+    let datasets = adequacyModes.map((mode, idx) => {
+        return {
+            backgroundColor: ADEQUACY_COLORS[mode],
+            label: getLegend(measurementMethod, mode),
+            data: censusGroups.map(_ => populationByAdequacyByGroup[_][idx])
+        }
+    })
+    let options
```

Comment:
Could you do a ternary on option maybe? Not sure what the best practice is here

Reply:
I thought about that too, but options is kinda big so I think it would get more confusing, right?